hibernate-reactive-core/src/test/java/org/hibernate/reactive/EagerElementCollectionTest.java

```diff
@@ -0,0 +1,275 @@
+/* Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * SPDX-License-Identifier: LGPL-2.1-or-later
+ * Copyright: Red Hat Inc. and Hibernate Authors
+ */
+package org.hibernate.reactive;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import javax.persistence.ElementCollection;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.Id;
+
+import org.hibernate.cfg.Configuration;
+import org.hibernate.reactive.stage.Stage;
+import org.hibernate.reactive.testing.DatabaseSelectionRule;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import io.vertx.ext.unit.TestContext;
+
+import static org.hibernate.reactive.containers.DatabaseConfiguration.DBType.POSTGRESQL;
+
+public class EagerElementCollectionTest extends BaseReactiveTest {
+
+	private Person thePerson;
+
+	@Rule
+	public DatabaseSelectionRule rule = DatabaseSelectionRule.runOnlyFor( POSTGRESQL );
+
+	protected Configuration constructConfiguration() {
+		Configuration configuration = super.constructConfiguration();
+		configuration.addAnnotatedClass( Person.class );
+		return configuration;
+	}
+
+	@Before
+	public void populateDb(TestContext context) {
+		String[] phoneArray = new String[]{ "999-999-9999", "111-111-1111", "123-456-7890" };
+		thePerson = new Person( 7242000, "Claude", phoneArray);
```

Comment:
Because the purpose of this test is to test the insert, I wouldn't put it in a `@Before` method. I would create a dedicated test.